management/tenant.go

```diff
@@ -105,6 +116,42 @@ type TenantFlags struct {
 	// If enabled, All your email links and urls will use your configured custom
 	// domain. If no custom domain is found the email operation will fail.
 	EnableCustomDomainInEmails *bool `json:"enable_custom_domain_in_emails,omitempty"`
+
+	// If enabled, users will not be prompted to confirm log in before SSO redirection.
+	EnableSso *bool `json:"enable_sso,omitempty"`
```

Comment:
Can we name this `EnableSSO` to more inline with Go best practices to capitalise abbreviations?

Reply:
yes !